src/Application/Commands/HandleOutOfSyncFunds.php

```diff
@@ -136,6 +140,15 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
             }
         }
 
+        $overmatchedDonations = $this->donationRepository->findOverMatchedDonations();
+        foreach ($overmatchedDonations as $donation) {
+            if (\in_array($donation->getUuid()->toString(), ['dcec4e60-8969-4f40-80f0-35bb5b7184af','c563fb10-ccbd-41d0-95c6-5a4e14596990'], true)) {
```

Comment:
I think maybe cleaner to decide how to patch these and do a migration next, but this LGTM for now